Formula/tcping.rb

```diff
@@ -1,8 +1,9 @@
 class Tcping < Formula
   desc "TCP connect to the given IP/port combo"
-  homepage "https://web.archive.org/web/20171102193822/linuxco.de/tcping/tcping.html"
-  url "https://mirrors.kernel.org/gentoo/distfiles/tcping-1.3.5.tar.gz"
-  sha256 "1ad52e904094d12b225ac4a0bc75297555e931c11a1501445faa548ff5ecdbd0"
+  homepage "https://github.com/mkirchner/tcping"
+  url "https://github.com/mkirchner/tcping/archive/1.3.6.tar.gz"
```

Comment:
Is the github repo the official repo for `tcping` now?

Reply:
From what I've seen, there wasn't a repo before, only [an index page of the archive files and some release folders](https://web.archive.org/web/20170923170357/http://linuxco.de/tcping/). The GitHub repo acts as everything for `tcping` now (homepage, release archive host, Git repo).

Reply:
Cool, I actually just checked `https://mirrors.kernel.org/gentoo/distfiles/tcping-1.3.6.tar.gz`, it does carry the same `sha256`.